Library/Homebrew/tap_auditor.rb

```diff
@@ -54,6 +56,9 @@ def audit_tap_formula_lists
       check_formula_list_directory "style_exceptions", @tap_style_exceptions
       check_formula_list "pypi_formula_mappings", @tap_pypi_formula_mappings
       check_formula_list ".github/autobump.txt", @tap_autobump
+
+      renames = @formula_renames.values # .reject { |name| @formula_renames.key?(name) || @tap_migrations.key?(name) }
+      check_formula_list "formula_renames", renames
```

Comment:
I guess it is probably fine to audit these and do one of following based on age of rename:
* remove older chained rename given migrator should have run (and warning would have been output for a while) or
* squash them to simplify handling if multiple renames happen over short time period (e.g. `a -> b` added 1 month ago and then `b -> c` so may want to keep original `a` until sufficient time period.)

```suggestion
      check_formula_list "formula_renames", @formula_renames.values
```